lms/djangoapps/certificates/management/commands/rescind_cert.py

```diff
@@ -0,0 +1,104 @@
+"""
+Management command to update the status of a single user certificate
+for a given user, in a given course. If no --status kwarg is provided,
+the status will be updated to CertificiateStatuses.unavailable.
+"""
+import logging
+from optparse import make_option
+
+from django.contrib.auth.models import User
+from django.core.management.base import BaseCommand, CommandError
+from opaque_keys import InvalidKeyError
+from opaque_keys.edx.keys import CourseKey
+from opaque_keys.edx.locations import SlashSeparatedCourseKey
+
+from certificates.models import GeneratedCertificate, CertificateStatuses
+
+LOGGER = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    '''Update the status of a single user certificate for a given user in a given course'''
+    help = u'Rescind a certificate for a particular user in a particular course.'
```

Comment:
For commands, I prefer this convention:

```python
class Command(BaseCommand):
    """
    Update the status of a single certificate for a given user/course combination
    """
    help = __doc__
```

This way, the help text is pulled directly from the `docstring`, which means there's one less string to keep in sync.

See how here the `docstring` text has already drifted away from the `help` text? One less thing to manage :)